README.md

```diff
@@ -1,39 +1,39 @@
-This is repository for the EDNS Workaround removal flag day!
+This is the repository for the EDNS workaround removal flag day!
```

Comment:
This line used to be meaningful when there was (almost) no other content, but now I'd think we should simply remove it.